tools/batchL2DataCreatedRollup/generateBatchL2DataCreatedRollup.sh

```diff
@@ -0,0 +1,2 @@
+npx hardhat run getData.ts --network zkevmDevnet
```

Comment:
Add a --network parameter in the bash script, si ti can be forwarded to the `npx hardhat run getData.ts --network $NETWORK`